Capfile

```diff
@@ -26,7 +26,8 @@ require 'dlss/capistrano'
 require 'capistrano/rvm'
 require 'whenever/capistrano'
 require 'capistrano/honeybadger'
-require 'capistrano/rails/migrations'
+require 'capistrano/rails'
```

Comment:
I think Erin and I were experimenting with getting the `listener:restart` task run by cap deploy and this snuck in here. I'll restore it.